### DISCUSSION TITLE
version 8.0 - new capabilities options

### DISCUSSION THREAD

**Discussion by RSM-SBA** [appium/java-client#1664] [Q&A] [answered]
I have switched my tests to java-client 8.0 and it worked well (thanks to the migration guide).
But i have a question to the newly introduced driver options.

How can/should i handle the capabilities for the flutter driver in future ?

**Comment by mykola-mokhnach** [comment, marked as answer]
You could either create a PR and describe available flutter driver capabilities as setters and getters or use the generic BaseOptions class with its `amend` method to add custom capabilities.

**Reply by RSM-SBA** [reply]
Thanks for your quick response.
As it looks like there are only 2 driver specific capabilities.
retryBackoffTime
maxRetryCount

I'm not a very good coder so i will not create a PR ;)

**Reply by mykola-mokhnach** [reply]
Nobody is ideal. We are happy to review PRs and add necessary comments. You could simply follow other driver options implementations as a template
